src/FixtureBuilder/ExpressionLanguage/Parser/TokenParser/Chainable/TolerantFunctionTokenParser.php

```diff
@@ -56,7 +56,7 @@ public function __construct(ChainableTokenParserInterface $functionTokenParser,
     /**
      * @inheritdoc
      */
-    public function withParser(ParserInterface $parser): self
+    public function withParser(ParserInterface $parser): parent
```

Comment:
🤔   I would remove the `: self` in the `AbstractChainableParserAwareParser` then